alacritty_terminal/src/input.rs

```diff
@@ -910,6 +918,15 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
         has_binding
     }
 
+    /// Set the cursor depending on where the mouse is on the message bar
+    fn message_cursor(&mut self, point: Point, message: Message) {
```

Comment:
The name `message_cursor` sounds like a Rust-style getter. Just renaming it to `update_message_cursor` would probably be fine here. Using `set_message_cursor` wouldn't be ideal since it's not actually a setter, so it doesn't take a mouse cursor as parameter.

Reply:
Yeah, that makes sense